src/v2/guide/class-and-style.md

```diff
@@ -202,3 +202,5 @@ Starting in 2.3 you can provide an array of multiple (prefixed) values to a styl
 ``` html
 <div :style="{ display: ['-webkit-box', '-ms-flexbox', 'flex'] }">
 ```
+
+This will only render the last value in the array which the browser supports. In this example, it will render `display: flex` for browsers that support the unprefixed version of flexbox.
```

Comment:
But doesn't it only set one value? For example, if you do `color: ['red', 'yellow', 'fake(value)']` it will only set `yellow`